pkg/rtcp/picture_loss_indication.go

```diff
@@ -11,6 +11,8 @@ type PictureLossIndication struct {
 
 	// SSRC where the loss was experienced
 	MediaSSRC uint32
+
+	header Header
```

Comment:
Good catch! This is left over from an earlier prototype. I switched to generating the header because it makes the packets directly comparable with `reflect.DeepEqual`